cluster/up.sh

```diff
@@ -31,7 +31,14 @@ fi
 
 ensure_golang
 
+## Validate
+operator_crds=$(./cluster/kubectl.sh get crd -l operator.v2v.kubevirt.io -o jsonpath={.items[*].metadata.name})
+check_structural_schema "${operator_crds[@]}"
```

Comment:
When was the `NonStructuralSchema` condition reported as `True`? What should be investigated when this validation fails?

Reply:
The k8s report that in case the CRD is not structural. In that case the CRD API is wrong, there are rules, which must comply to have structural schema: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#specifying-a-structural-schema

Reply:
So it looks like it boils down to fixing something in https://github.com/kubevirt/vm-import-operator/blob/master/pkg/operator/resources/operator/operator.go. Is that correct?

Reply:
Right, in one of `resourcemappings.v2v.kubevirt.io`, `virtualmachineimports.v2v.kubevirt.io`, `vmimportconfigs.v2v.kubevirt.io`.